ci/ubuntu-24.04/Dockerfile

```
@@ -47,6 +48,10 @@ RUN apt-get update && apt-get -y install \
 RUN pip3 install --break-system-packages websockets junit2html
 RUN gem install coveralls-lcov
 
+# Ubuntu installs clang versions with the binaries having the version number
+# appended. Create a symlink for clang-tidy so cmake finds it correctly.
+RUN if [ ! -f /usr/bin/clang-tidy ]; then ln -s /usr/bin/clang-tidy-19 /usr/bin/clang-tidy; fi
```

Maybe update-alternatives?

```suggestion
RUN update-alternatives --install /usr/bin/clang-tidy clang-tidy /usr/bin/clang-tidy-19 1000
```